environment/src/main/resources/schema/app/20200323193238_CB-5159_freeipa_instances_per_group_is_optional.sql

```diff
@@ -0,0 +1,10 @@
+-- // CB-1617 freeipa creation is optional
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE environment ADD COLUMN IF NOT EXISTS freeIpaInstancesPerGroup integer NOT NULL DEFAULT 1;
```

Comment:
please don't use `DEFAULT`, because it can lock the table. instead you can use an update